### PR TITLE
add foundit blog post

### PR DESCRIPTION
**Pull request by rickywid**
my blog post contribution.

### CODE REVIEW

**Comment by rickywid** [comment]
I accidentally installed the react-burger-menu in this branch. I uninstalled the package but the package.lock.json file is still modified. Will this be an issue?

File: content/author.yaml [hunk 1]
```diff
@@ -8,3 +8,8 @@
   bio: Core Team Member at Project Unicorn
   github: roymoran
   avatar: ./assets/rmoran.jpeg
+- id: rickyjw
+  name: Ricky JW
+  bio: Core Team Member at Project Unicorn
+  github: rickywid
+  avatar: ./assets/profile1.png
```
Comment (rmjordas) [inline content/author.yaml]: Rename `profile1.png` -> `rickywid.png` and change this line too.

File: content/blog/2019-08-31-foundit-project-retrospective/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+title: 'FoundIT Retrospective'
+author: rickyjw
+date: '2019-08-31'
+image: ./hero.png
+tags:
+  - Project Unicorn
+---
+
+In this post I will talk about my experience building my new project [FoundIT](http://foundit.herokuapp.com) and things that I've learned as being a team lead for the first time.
```
Comment (rmjordas) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]: [a 1/2] Use HTTPS in the FoundIT link. Search engine crawlers can penalize sites that contain `http` links.

```suggestion
In this post I will talk about my experience building my new project [FoundIT](https://foundit.herokuapp.com) and things that I've learned as being a team lead for the first time.
```

File: content/blog/2019-08-31-foundit-project-retrospective/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+### What Is Project Unicorn
+
+[Project Unicorn](https://projectunicorn.net/) is an online programming community where developers can collaborate together to build applications. Our goal is to bring the development community together where everyone can learn, develop and improve their skills by building meaningful projects together with others who share the same passion.
+
+### FoundIT Overview
+
+[FoundIT](http://foundit.herokuapp.com) is a web application where users can post lost or found items online in hopes of returning it back to its rightful owner. Everybody hates that feeling of reaching into their pockets only to find out that their brand new IPhone X or Gucci wallet has gone missing. Millions of valuable personal items is lost everyday and the development team came up with the idea to create a dedicated website to help solve this issue.
```
Comment (rmjordas) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]: [a 2/2] Use HTTPS in the FoundIT link...
```suggestion
[FoundIT](https://foundit.herokuapp.com) is a web application where users can post lost or found items online in hopes of returning it back to its rightful owner. Everybody hates that feeling of reaching into their pockets only to find out that their brand new IPhone X or Gucci wallet has gone missing. Millions of valuable personal items is lost everyday and the development team came up with the idea to create a dedicated website to help solve this issue.
```

Comment (rmjordas) [reply content/blog/2019-08-31-foundit-project-retrospective/index.md]: Also:
- IPhone -> iPhone

File: content/blog/2019-08-31-foundit-project-retrospective/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+- [Cloudinary](https://cloudinary.com/documentation/cloudinary_references)
+- [Google Maps](https://developers.google.com/maps/documentation/javascript/tutorial)
+- [IPData](https://ipdata.co/)
+
+#### Project Management/Communication Tools
+
+- [Slack](https://slack.com)
+- [Google Docs](https://docs.google.com)
+- [Google Hangouts](https://hangouts.google.com/webchat/start)
+- [Github Project Board](https://github.com/features/project-management/)
```
Comment (rmjordas) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]: [b 1/2] Github -> GitHub
```suggestion
- [GitHub Project Board](https://github.com/features/project-management/)
```

File: content/blog/2019-08-31-foundit-project-retrospective/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+- Ability to post, view and search Lost or Found missing items
+- Option to upload an image of the item
+- Detect user’s location using geolocation
+- Display a map of where the item was lost or found
+- Focus on a clean simple to use UI and responsive design
+
+The list of features was small but it was enough to cover the core functionality of our web application.
+
+Setting up the front end side of the application was effortless for the most part. The challenging tasks was setting up the server side code. Nobody on the team had any experience with setting up continuous integration/deployment (CI/CD) or any of the other technologies used in the back end. This meant countless hours on youtube, stackoverflow, blogs, books, documentation and any other sources of information we could get our hands on to help piece the puzzle together. No doubt about it, some of our developers gained a few extra grey hairs.
```
Comment (rmjordas) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]: - youtube -> YouTube
- stackoverflow -> Stack Overflow

File: content/blog/2019-08-31-foundit-project-retrospective/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+In order to keep the project moving forward, we used [Github's Project Board](https://github.com/features/project-management/) to track and manage our tasks. This was a great alternative to similar project management tools such as [Jira](https://www.atlassian.com/software/jira). Assigning specific tasks to individual members kept everyone accountable for their own work but it also allowed team members know what everyone else was currently working on at the same time.
```
Comment (rmjordas) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]: [b 2/2] Github -> GitHub
```suggestion
In order to keep the project moving forward, we used [GitHub's Project Board](https://github.com/features/project-management/) to track and manage our tasks. This was a great alternative to similar project management tools such as [Jira](https://www.atlassian.com/software/jira). Assigning specific tasks to individual members kept everyone accountable for their own work but it also allowed team members know what everyone else was currently working on at the same time.
```

File: content/blog/2019-08-31-foundit-project-retrospective/index.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+### How Project Unicorn has helped
+
+One of the advantages of being part of Project Unicorn is the community. We are all here to support each other. Project Unicorn is a place where we foster learning and growth. There are so many knowledgeable and experienced developers ranging from front end, back end, devops and project management. Their expertise helped me during FoundIT's development and without their help and guidance I don’t know if I would have had the energy and motivation to continue on. Project Unicorn is still a new community and we are growing every single day.
+
+Whether you’re a beginner who wants to move away from tutorials or that seasoned veteran who’s been building projects for 10 plus years, Project Unicorn is a great community to be part of.
+
+If you would like to learn more about Project Unicorn then check out these related blog posts
+
+- [Project Unicorn UI](http://localhost:8000/blog/project-unicorn-ui/)
```
Comment (rmjordas) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]:
```suggestion
- [Project Unicorn UI](/blog/project-unicorn-ui/)
```

File: content/blog/2019-08-31-foundit-project-retrospective/index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+- [Contributing Part I: Getting Started](https://projectunicorn.net/blog/contributing-part-i-getting-started/)
```
Comment (rmjordas) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]:
```suggestion
- [Contributing Part I: Getting Started](/blog/contributing-part-i-getting-started/)
```

**Comment by rmjordas** [comment]
> I accidentally installed the react-burger-menu in this branch when I was testing it. I uninstalled the package but the package.lock.json file is still modified. Will this be an issue?

@rickywid Not sure but to be safe you can replace this current `package.lock.json` with the one from the [parent commit](https://github.com/projectunic0rn/pub/blob/753670bf9e79e7f16bcdfa62c5faf9ac39b5fffe/package-lock.json) then push that commit to this branch.

File: content/blog/2019-08-31-foundit-project-retrospective/index.md (hunk 3 above)
Comment (aaron2212) [inline content/blog/2019-08-31-foundit-project-retrospective/index.md]: `millions of valuable personal items is lost` -> `millions of valuable personal items are lost`

File: .env.example [hunk 9]
```diff
@@ -1,2 +0,0 @@
-GATSBY_TELEMETRY_DISABLED=1
-GA_TRACKING_ID="UA-123456789-0"
```
Comment (rmjordas) [inline .env.example]: Please restore this file.

**Comment by rickywid** [comment]
@rmjordas i think the file should be restored?

**Comment by rmjordas** [comment]
> @rmjordas i think the file should be restored?

@rickywid Are you talking about `package-lock.json`? Yeah, it should be restored. 👍

**Comment by rickywid** [comment]
> > @rmjordas i think the file should be restored?
> 
> @rickywid Are you talking about `package-lock.json`? Yeah, it should be restored.

I mean the .env.example file

**Comment by rmjordas** [comment]
> I mean the .env.example file

@rickywid Oh. Yeah, but you already restored `.env.example` so I thought you're talking about `package-lock.json`. 😄  Anyway, both changes to `.env.example` and `package-lock.json` are out of scope for this pull request i.e., adding a blog post, so I suggested to change them back.

**Comment by rickywid** [comment]
> > I mean the .env.example file
> 
> @rickywid Oh. Yeah, but you already restored `.env.example` so I thought you're talking about `package-lock.json`.  Anyway, both changes to `.env.example` and `package-lock.json` are out of scope for this pull request i.e., adding a blog post, so I suggested to change them back.

@rmjordas Ok since these were out of the scope of the for this pull request, should I revert the package-lock.json file and .env.example file?

**Comment by rmjordas** [comment]
> @rmjordas Ok since these were out of the scope of the for this pull request, should I revert the package-lock.json file and .env.example file?

@rickywid You already restored `.env.example` and changed the content of `package-lock.json` to the previous version from what I can tell, except whitespaces are removed. See images below.

packagelockjsonvscode

packagelockjson

**Comment by rickywid** [comment]
> > @rmjordas Ok since these were out of the scope of the for this pull request, should I revert the package-lock.json file and .env.example file?
> 
> @rickywid You already restored `.env.example` and changed the content of `package-lock.json` to the previous version from what I can tell, except whitespaces are removed. See images below.
> 
> packagelockjsonvscode
> 
> packagelockjson

@rmjordas  Ok is there anything else I have to do to get this PR approved?

**Comment by rmjordas** [comment]
> @rmjordas Ok is there anything else I have to do to get this PR approved?

@roymoran and @aaron2212 are the ones that can approve or reject pull requests. I only add comments/suggestions. 😄 

But there's one more thing. You renamed your avatar image file but `author.yaml` still contains:

```yml
  avatar: ./assets/profile1.png
```

Finally, I forgot to add a note on why I suggested using relative paths on the links to other blog posts. Using relative paths allows the app to navigate to the other posts with no page refresh whereas using full URLs will open them in another tab (I believe). 

Depending on what you want, you can use either.

**Comment by rickywid** [comment]
> > @rmjordas Ok is there anything else I have to do to get this PR approved?
> 
> @roymoran and @aaron2212 are the ones that can approve or reject pull requests. I only add comments/suggestions. 
> 
> But there's one more thing. You renamed your avatar image file but `author.yaml` still contains:
> 
> ```yaml
>   avatar: ./assets/profile1.png
> ```
> 
> Finally, I forgot to add a note on why I suggested using relative paths on the links to other blog posts. Using relative paths allows the app to navigate to the other posts with no page refresh whereas using full URLs will open them in another tab (I believe).
> 
> Depending on what you want, you can use either.

@rmjordas oh I thought you had permissions to approve PR's. I pushed some more changes. Let me know if everything looks good.

**Comment by rmjordas** [comment]
@rickywid I mean, I can approve PRs using GitHub's interface, but @roymoran and @aaron2212 have the final say.

LGTM 👍

**Comment by roymoran** [comment]
@rmjordas thanks for reviewing this pr.

**Pull request merged**